sdk/go/auto/workspace.go

```diff
@@ -92,6 +92,8 @@ type Workspace interface {
 	ListStacks(context.Context) ([]StackSummary, error)
 	// InstallPlugin acquires the plugin matching the specified name and version.
 	InstallPlugin(context.Context, string, string) error
+	// InstallPluginFromServer acquires the plugin matching the specified name and version.
+	InstallPluginFromServer(context.Context, string, string, string) error
```

Comment:
I wonder if InstallPluginWithOpts would be a better extension? There's a load of other options like --checksum --reinstall --exact --file that we're not currently exposing.

Reply:
That probably makes sense but is likely outside my wheelhouse/capabilities to implement

Reply:
hah fair, this all needs an overhaul in the future this is a good improvement for now.